src/server.ts

```diff
@@ -22,12 +22,27 @@ export function getBrokerUrl({
     sourceUrl.pathname = sourceIndex;
   }
 
-  const brokerUrl = new URL('file://');
-  brokerUrl.pathname = path.resolve(__dirname, '../broker/index.html');
-  const hashParam = encodeHashParameter({
-    src: sourceUrl.href,
-    loadMode,
-    ...outputSize,
-  });
-  return `${brokerUrl.href}#${hashParam}`;
+  const viewerUrl = new URL('file://');
+  viewerUrl.pathname = upath.join(
+    resolvePkg('@vivliostyle/viewer', { cwd: __dirname })!,
+    'lib/index.html',
+  );
+
+  const pageSizeValue =
+    outputSize &&
+    ((outputSize as { format: string }).format ||
+      `${(outputSize as { width: string }).width} ${
+        (outputSize as { height: string }).height
+      }`);
```

Comment:
nits: as castingが無いほうがよりスマートそうです
```suggestion
  const pageSizeValue =
    outputSize &&
    ('format' in outputSize
      ? outputSize.format
      : `${outputSize.width} ${outputSize.height}`);
```

Reply:
なるほど、そのように書けるのですね。ありがとうございます。